### PR TITLE
US- 2310 Postback 

### PR DESCRIPTION
**Pull request by bhoomraogaripriyanka**
Post back functionality

### CODE REVIEW

File: panda-qa/Resources/CommonFunctionality.robot
Comment (AkhileshKhanduri25) [inline panda-qa/Resources/CommonFunctionality.robot:1]: working fine for me.

File: panda-qa/Resources/PageObjects/BarronsCapabilitiesPage.robot
Comment (AkhileshKhanduri25) [inline panda-qa/Resources/PageObjects/BarronsCapabilitiesPage.robot:1]: page should contain Wait Until Element Is Visible  //h1[text()="Barrons" as per the Button value

File: panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot
Comment (AkhileshKhanduri25) [inline panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot:1]: This need to checked and changed accordingly as still Page Should Contain Element  //div[@class="byline"]/div/div/span[text()="Catey Hill"] but Catey Hill is not getting located.

File: panda-qa/Tests/FollowWidget/Regression/WSJ_Regression.robot
Comment (AkhileshKhanduri25) [inline panda-qa/Tests/FollowWidget/Regression/WSJ_Regression.robot:1]: Element with locator '//*[text()="Dave Michaels"]' is still not getting found.

File: panda-qa/Resources/PageObjects/MarketWatchMailboxPage.robot
Comment (AkhileshKhanduri25) [inline panda-qa/Resources/PageObjects/MarketWatchMailboxPage.robot:1]: Code is working fine for MarketWatch Mailbox Page.

Comment (AkhileshKhanduri25) [inline panda-qa/Resources/PageObjects/MarketWatchMailboxPage.robot:1]: Unsubscribe button is not reflecting but code is working fine for marketwatch mailbox page.

Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/MarketWatchMailboxPage.robot:1]: Sometimes the unsubscribe button takes time to reflect ,  expected functionality

File: panda-qa/Resources/PageObjects/BarronsCapabilitiesPage.robot
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/BarronsCapabilitiesPage.robot:1]: This is not part of post back functionality

File: panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot
Comment (bhoomraogaripriyanka) [reply panda-qa/Resources/PageObjects/MarketWatchArticlePage.robot:1]: This does not cover post back functionality

File: panda-qa/Tests/FollowWidget/Regression/WSJ_Regression.robot
Comment (bhoomraogaripriyanka) [reply panda-qa/Tests/FollowWidget/Regression/WSJ_Regression.robot:1]: This is not part of post back functionality

**Review by ArchanaSaini149 — APPROVED** [review]

**Pull request merged**